### ISSUE TITLE
BAD_PRACTICE when throwing sub classes of java.lang.Error

### ISSUE THREAD

**Issue by afrin216** [mebigfatguy/fb-contrib#194]
What is your opinion on adding a detector under `BAD_PRACTICE` category for applications that
* throw `java.lang.Error` or its sub-classes
* try to catch/handle `java.lang.Error` or its sub-classes

Thanks
Afrin

**Comment by mebigfatguy** [comment]
I agree catching error is wrong. Not sure about the throwing part. I'm OK with that in some cases
